src/index.js

```diff
@@ -18,15 +18,42 @@ import Player from './Player';
 import Round from './Round';
 import Turn from './Turn';
 
+function createPlayers(Player1, Player2, Player3) {
+  let player1 = new Player(Player1, 1)
+  let player2 = new Player(Player2, 2)
+  let player3 = new Player(Player3, 3)
```

Comment:
Why are you passing a second argument on in the new instance, I believe the parameter is score for that argument which defaults to 0.